lib/cartopy/crs.py

```diff
@@ -3219,6 +3219,42 @@ def x_limits(self):
     def y_limits(self):
         return self._y_limits
 
+class Spilhaus(Projection):
+    """
+    Spilhaus World Ocean Map in a Square
+
+    """
+    def __init__(self,rotation = 45, false_easting = 0.0, false_northing = 0.0):
```

Comment:
The other classes in this module use a default `globe=None` which then allows the datum to be overridden.

It also looks like every other class has a `central_longitude` parameter as well, but it is always 0 in the other projections and would be 66.94970198 here to correspond to the default lon0 in PROJ. So I'm not sure whether we want to expose that parameter or not for consistency. I can go either way on this one, so fine with how you have it for now.

```suggestion
    def __init__(self, rotation=45, false_easting=0.0, false_northing=0.0, globe=None):
```

Reply:
There are some linter that can change this kind of formatting. I can add it to .pre-commit-config (not sure if I am spelling this correctly) if you want. 

I will add the globe argument, sorry for the ommision.